packages/patternfly-4/react-docs/src/html.js

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+export default function HTML(props) {
+  return (
+    <html {...props.htmlAttributes}>
+      <head>
+        <meta charSet="utf-8" />
+        <meta httpEquiv="x-ua-compatible" content="ie=edge" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+        {props.headComponents}
+      </head>
+      <body {...props.bodyAttributes} className="pf-m-redhat-font">
```

Comment:
It's possible to globally include SASS that adds that styling to `body`, but I like this better to give us more control of the template in the future.

Reply:
I think for now we don't really need to add a template here, this docs site isn't going to last forever anyway once we can move to just using pf.org.